### PR TITLE
Create content component

### PR DESCRIPTION
**Pull request by Vacariuc-Alex**

### CODE REVIEW

File: src/components/navbar/FormDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+
+    const handleOnPriceChange = (e: any) => {
+        let price = e.target.value;
+
+        // Remove all characters that are not digits or decimal separators
+        price = price.replace(/[^\d.,]/g, '');
+
+        // Replace comma with period to standardize decimal separators
+        price = price.replace(/,/g, '.');
+
```
Comment (sergheistratu) [inline src/components/navbar/FormDialog.tsx]: why few times price ?

File: src/utils/helpers/schemes.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    model: string().required(),
+    year: number().required().min(1900).max(new Date().getFullYear()),
+    vin: string().required().matches(/[A-HJ-NPR-Z0-9]{17}/, "Invalid VIN format\nExample of valid VIN: 4Y1SL65848Z411439"),
+    color: string().required(),
+    price: number().required().positive(),
+    images: array().of(
+        string().test({
+            message: (value) => {
+                const index = value.path.split(/\[|\]/)[1];
+                console.log(index)
```
Comment (sergheistratu) [inline src/utils/helpers/schemes.js]: remove console

**Review by sergheistratu — APPROVED** [review]

File: src/components/navbar/FormDialog.tsx (hunk 1 above)
Comment (Vacariuc-Alex) [reply src/components/navbar/FormDialog.tsx]: fixed

File: src/utils/helpers/schemes.js (hunk 2 above)
Comment (Vacariuc-Alex) [reply src/utils/helpers/schemes.js]: fixed

**Pull request merged**